docker-readme.md

```diff
@@ -0,0 +1,27 @@
+# Build docker:
```

Comment:
Let's keep this change to just the bug fix. The docker changes can be done in the other pull request.

Reply:
sry, messed with commits. Gonna fix it.